projects/core/components/tooltip/tooltip-options.ts

```diff
@@ -0,0 +1,30 @@
+import {InjectionToken} from '@angular/core';
+import {TuiDirection, TuiHintModeT} from '@taiga-ui/core/types';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface TooltipOptions {
```

Comment:
Everything, except for the icon is actually more relevant to `HintDirective`. How about we move it there and keep the icon there as well? I know icon is irrelevant for the directive, but I think it better to have a single source of default values for both `Tooltip` and `Hint` and making a separate token just for the icon might be an overkill.

Reply:
I'm not sure this is the best solution that I've come up with. Check the last commit please, I'm sure there is a better way.